content/blog/2021/support-unknown.md

```diff
@@ -17,7 +17,7 @@ summary: |
   a way for developers to start integrating the new code,
   without breaking their sites on legacy browsers.
   That looks different depending on the feature,
-  but can often include new tests 
```

Comment:
Interesting. I thought I had a setting that would handle this…

Reply:
```
  "files.trimTrailingWhitespace": true,
```